extensions/panache/hibernate-orm-panache-rest/runtime/src/main/java/io/quarkus/hibernate/orm/panache/rest/PanacheCrudController.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.hibernate.orm.panache.rest;
+
+import javax.ws.rs.core.Response;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+
+public interface PanacheCrudController<T extends PanacheEntityBase, ID> {
+
+    /**
+     * Return all entities as a JSON array.
+     * Response content type: application/json.
+     *
+     * @return A response with a JSON array of all entities.
+     */
+    Response list();
```

Comment:
Non-paginated list is very bad design, so maybe we should provides a paginated version by default ?

Reply:
For sure. I'll add paging, sorting and limiting once we're done with the initial PR.